abr_control/controllers/osc.py

```diff
@@ -78,30 +79,46 @@ def generate(self, q, dq,
             current joint angles [radians]
         dq : float numpy.array
             current joint velocities [radians/second]
-        target_pos : float numpy.array
-            desired joint angles [radians]
-        target_vel : float numpy.array, optional (Default: numpy.zeros)
-            desired joint velocities [radians/sec]
+        target: 6 dimensional float numpy.array
+            desired task space position and orientation [meters, radians]
+        target_vel : float 6D numpy.array, optional (Default: None)
+            desired task space velocities [meters/sec, radians/sec]
```

Comment:
shouldn't this be just rad/sec? m/s would be if we were setting the desired velocity in task space.

Reply:
this is in task space